Neos.Flow/Classes/Persistence/Doctrine/Repository.php

```diff
@@ -161,6 +161,8 @@ public function findAllIterator(): IterableResult
      * @param IterableResult $iterator
      * @param callable|null $callback
      * @return \Generator
+     *
+     * @deprecated Will be removed with Flow 9. The {@see findAllIterator} will return an \iterable directly
      */
     public function iterate(IterableResult $iterator, callable $callback = null): ?\Generator
```

Comment:
nice :D thx for removing this stuffy. Fyi we also have to remove it from the thumbnail repository and others in Neos.Neos ;)

Reply:
Yep I know, I know! I would add this to the respective Neos 9 dbal changes then, but first wanted to get this in to be sure I _can_ actually remove it 😆